models/media_stats/snowplow_media_player_media_stats.sql

```diff
@@ -31,6 +31,7 @@ with new_data as (
 
   select
     p.media_identifier,
+    p.app_id,
```

Comment:
Same with media ads. Let's remove to avoid a breaking change or check or agree to change the structure fully despite not needed the app not currently requiring it.